pages/fundamentals/introduction.html

```diff
@@ -1,190 +1,185 @@
-﻿<h1>An Introduction of Go</h1>
+﻿<h1>Go 소개</h1>
 
 <p>
-Go is a compiled and static typed programming language born from Google.
-Many of the core Go design and tool development team members have many years
-of experience in the field of programming language research.
+Go는 구글에서 만든 컴파일된 정적 타입 프로그래밍 언어 입니다.
+Go 디자인 및 도구 개발 팀의 많은 핵심 팀원들은 프로그래밍 언어 연구 분야에서 수년간의 경험을 가지고 있습니다.
 </p>
 
 <div>
-Go has many features. Some are unique, some are borrowed from other programming languages:
+Go는 많은 기능이 있습니다. 일부는 고유하게 작성된 것이고 일부는 다른 언어에서 차용하였습니다
 <ul>
 <li>
-	built-in concurrent programming support
+	내장 동시 프로그래밍 지원
 	<ul>
 	<li>
-		goroutines (green threads) and start new goroutines easily.
+		고루틴(녹색 글타래)을 사용하여 새 고루틴을 쉽게 시작할 수 있습니다.

```

Comment:
프로세스 스레드 할 때의 thread 같은데 일단 글타래로는 번역하지 않는게 나을 것 같습니다.
green thread를 뭐로 옮길지는 잘 모르겠네요. 지금 처음 드는 생각으로는 그냥 그린 스레드로 쓰는 게 좋아 보입니다.

원문도 좀 이상한데 그냥 다음과 같이 써 버리는 것도 나쁘지 않을 것 같습니다. 다른 분들도 의견 주시면 좋겠습니다.
```suggestion
		쉽게 사용할 수 있는 그린 스레드인 고루틴
```

Reply:
수정했습니다